testing/coreidle/test_main_c.cpp

```diff
@@ -399,6 +394,36 @@ TEST_P(coreidle_main_c_p, parse_err2) {
   EXPECT_EQ(coreidle_main(13, argv), 0);
 }
 
+/**
+ * @test       invalid_main*
+ * @brief      Test: coreidle_main
+ * @details    When called with parameters will nullbytes,<br>
+ *             coreidle_main runs to completion, it fails to read <br>
+ *             .gbs file and returns zero upon successful cleanup.<br>
+ */
+TEST_P(coreidle_main_c_p, invalid_main0) {
+  char zero[32];
+  char one[32];
+  char two[32];
+  strcpy(zero, "coreidle");
+  strcpy(one, "-G");
+  strcpy(two, "temp\0.gbs");
+
+  char *argv[] = { zero, one, two };
+  EXPECT_EQ(coreidle_main(3, argv), 0);
```

Comment:
two will contain { 't', 'e', 'm', 'p', '\0', '.', 'g', 'b', 's', '\0' }, which is equivalent to the string "temp", a valid file name.  Not sure what this is testing.